### PR TITLE
update the way people write entries

### PR DESCRIPTION
**Pull request by sana7044**
update the way people write entries

### CODE REVIEW

File: src/components/Admin/TableEntry.vue [hunk 1]
```diff
@@ -1,45 +1,58 @@
...
+    :rows="rows"
+    :title="!userStore.isEditorOrAbove ? 'Manage Entries' : ''"
+  >
     <template v-slot:body-cell-actions="props">
       <td class="text-right">
         <q-btn
-          v-if="props.row.isWinner != true"
-          color="black"
-          :disable="userStore.getUser.role !== 'Admin'"
+          v-if="props.row.isWinner != true && userStore.isEditorOrAbove"
```
Comment (marclupanc) [inline src/components/Admin/TableEntry.vue:20]: Comparison props.row.isWinner != true may cause unexpected type coercion. Maybe replace it with "!=="

File: src/components/Admin/TableEntry.vue [hunk 2]
```diff
@@ -1,45 +1,58 @@
...
+          color="black"
           size="sm"
           icon="toggle_off"
+          :disable="userStore.getUser.role !== 'Admin'"
           @click="onSelectWinnerDialog(props.row)"
         >
           <q-tooltip class="positive" :offset="[10, 10]">select winner!</q-tooltip>
         </q-btn>
         <q-btn
           v-if="props.row.isWinner == true"
```
Comment (marclupanc) [inline src/components/Admin/TableEntry.vue:31]: Maybe replace as above.

File: src/components/shared/TheEntries.vue [hunk 3]
```diff
@@ -4,22 +4,36 @@
     :data-test="props.entries ? 'entries' : ''"
     style="padding-bottom: 7rem"
   >
-    <h2 class="q-my-auto text-bold text-h5 q-pa-md">Entries</h2>
+    <h2 class="q-my-auto text-bold text-h5 q-pa-md">
+      Entries
+      <q-icon name="add" size="30px" @click="openEntryDialog()"><q-tooltip>Add entries</q-tooltip></q-icon>
```
Comment (marclupanc) [inline src/components/shared/TheEntries.vue:9]: I would suggest adding "cursor: pointer" to the + button for better user experience

**Review by marclupanc — CHANGES_REQUESTED** [review]
Nice PR. Some minor suggestions from me 👍

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**